src/main/java/LogoutButton.java

```diff
@@ -0,0 +1,19 @@
+import coursescheduler.PanelManagerI;
```

Comment:
Delete file and create task for one button to take the user back to the login page to avoid code duplication.